alfasim_sdk/types.py

```diff
@@ -44,24 +46,27 @@ class String(BaseField):
 
 
     """
-    value: str = attrib(validator=instance_of(str))
+    value: str = attrib(validator=[instance_of(str), check_string_is_not_empty])
 
 
 @attr.s(kw_only=True)
 class Enum(BaseField):
-    value: List[str] = attrib()
+    values: List[str] = attrib()
     initial: str = attrib(validator=optional(instance_of(str)), default=None)
 
-    @value.validator
-    def check(self, attr, values):
-        if not isinstance(values, list):
-            raise TypeError(f"{attr.name} must be a list, got a {type(values)}.")
+    @values.validator
+    def check(self, attr, user_values):
```

Comment:
I'm curious why you changed `values` to `user_values`... `values` matches the name of the attribute we are validating after all. 🤔

Reply:
But it clashes with the decorator ` @values.validator` and I thought that this could be a problem.
Should I change?

Reply:
Oh they definitely don't interfere with each other, please change it. :